distribution/lib/Standard/Base/0.0.0-dev/src/Enso_Cloud/Enso_Secret.enso

```diff
@@ -31,7 +31,19 @@ polyglot java import org.enso.base.enso_cloud.HideableValue.SecretValue
 ## A reference to a secret stored in the Enso Cloud.
 type Enso_Secret
     ## PRIVATE
-    Value name:Text id:Text path:Enso_Path
+    private Value internal_name:Text id:Text internal_path:Enso_Path
+
+    ## GROUP Metadata
+         ICON metadata
+         The name of the secret.
+    name : Text
+    name self = self.internal_name
+
+    ## GROUP Metadata
+       ICON metadata
+       The path of the secret.
+    path : Enso_Path
+    path self = self.internal_path
```

Comment:
```suggestion
    ## GROUP Metadata
       ICON metadata
       The path of the secret.
    path : Text
    path self = self.internal_path.to_text
```

The `Enso_Path` is an internal `private` type and should not be exposed to users. Let's return text instead. That's also consistent with `Enso_File`.

Reply:
Have applied - hadn't changed as was just privatizing constructor.